doc/nrf/releases/release-notes-latest.rst

```diff
@@ -20,11 +20,16 @@ See the change log for each library in the :doc:`nrfxlib documentation <nrfxlib:
 sdk-zephyr
 ==========
 
-The Zephyr fork in |NCS| contains all commits from the upstream Zephyr repository up to and including ``ac117c8d``, plus some |NCS| specific additions.
+The Zephyr fork in |NCS| contains all commits from the upstream Zephyr repository up to and including ``4ef29b34e3``, plus some |NCS| specific additions.
 
 The following list summarizes the most important changes inherited from upstream Zephyr:
 
-*
+Kernel
```

Comment:
This format doesn't work with this introduction - if we're using subsections, this is not a list anymore. And also, then we cannot continue with a list of NCS-specific additions (because that would be part of the last subsection).
So we can either use nested lists here or change the structure/introduction.

Reply:
I'd really rather use sub-sections instead of nested lists to be honest. These lists risk becoming longer in the future, and I think it would be more readable for users to have the changes separated into different sub-sections.